actors/common/src/main/java/org/sunbird/userorg/UserOrgService.java

```diff
@@ -0,0 +1,226 @@
+        headers.put(AUTHORIZATION, "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJlMDRkNzJkMWNiZDg0MTEyOTBkNGFiZWM3NDU5YTFlYiJ9.bThu42m1nPTMikbYGywqBqQYUihm_l1HsmKMREMuSdM");
+        headers.put("Content-Type", "application/json");
+        return headers;
+    }
+
+    @Override
+    public Response getOrganisationDetails(String orgID)
+    {
+        Map<String, Object> requestMap = new HashMap<>();
+        Map<String, Object> request = new HashMap<>();
+        requestMap.put(JsonKey.REQUEST, request);
+        Map<String, String> filterlist = new HashMap<>();
+        request.put(FILTERS, filterlist);
+        filterlist.put("id",orgID);
+        Map<String, String> headers = getdefaultHeaders();
+        ObjectMapper mapper=new ObjectMapper();
+
+        Response response = null;
+        try {
+            String requestUrl =
+                    "https://dev.sunbirded.org/api"
+                            + getConfigValue(SUNBIRD_GET_ORGANISATION_API);
+            String reqBody = mapper.writeValueAsString(requestMap);
+
+            HttpResponse<String> httpResponse = Unirest.post(requestUrl).headers(headers).body(reqBody).asString();
+
+            if (StringUtils.isBlank(httpResponse.getBody())) {
+                throwServerErrorException(
+                        ResponseCode.SERVER_ERROR, errorProcessingRequest.getErrorMessage());
+            }
+            response = mapper.readValue(httpResponse.getBody(), Response.class);
+            if (!ResponseCode.OK.equals(response.getResponseCode())) {
+
+                throw new ProjectCommonException(
+                        response.getResponseCode().name(),
+                        response.getParams().getErrmsg(),
+                        response.getResponseCode().getResponseCode());
+            }
+        } catch (IOException e) {
+            log(
+                    "Exception occurred with error message = "
+                            + e.getMessage(),
+                    e);
+            throwServerErrorException(ResponseCode.SERVER_ERROR);
+        } catch (UnirestException e) {
+            log(
+                    "Exception occurred with error message = "
+                            + e.getMessage(),
+                    e);
+            throwServerErrorException(ResponseCode.SERVER_ERROR);
+        }
+        return response;
+    }
+
+    @Override
+    public Response getOrganisationDetailsForMultipleOrgIds(List<String> orgfields)
+    {
+        Map<String, Object> requestMap = new HashMap<>();
+        Map<String, Object> request = new HashMap<>();
+        requestMap.put(JsonKey.REQUEST, request);
+        Map<String, List<String>> filterlist = new HashMap<>();
+        request.put(FILTERS, filterlist);
+        filterlist.put("id",orgfields);
+        Map<String, String> headers = getdefaultHeaders();
+        ObjectMapper mapper=new ObjectMapper();
+
+        Response response = null;
+        try {
```

Comment:
same code is copied at multiple place, create a common method which accepts input as relative url, map as request and map as headers.

this common method need to be used by two other method one for get another for post. and those two methods need to be consumed further in other methods